README.rst

```diff
@@ -1,5 +1,35 @@
-Installation of Unicore CMS
-===========================
+A4W Style Guide
+================================
+
+For a general overview of Unicore's WoW, see our `WoW Documentation <http://ways-of-working.readthedocs.org/en/latest/process.html#our-front-end-development-process>`_. Overall, we use an approach known as Progressive Enhancement.
```

Comment:
Might be worth mentioning that these are Praekelt's Ways of Working, using the full words rather than the acronym?